packages/api-clients/open-api/bffApi.yml

```diff
@@ -14424,6 +15033,123 @@ components:
           format: uuid
       required:
         - id
+    DelegationState:
+      type: string
+      description: Delegation State
+      enum:
+        - WAITING_FOR_APPROVAL
+        - ACTIVE
+        - REJECTED
+        - REVOKED
+    DelegationTenant:
+      type: object
+      additionalProperties: false
+      properties:
+        id: 
+          type: string
+          format: uuid
+        name: 
+          type: string
+      required:
+        - id
+        - name
+    DelegationEService:
+      type: object
+      additionalProperties: false
+      properties:
+        id: 
+          type: string
+          format: uuid
+        name: 
+          type: string
+        description:
+          type: string
+      required:
+        - id
+        - name
+    Delegation:
+      type: object
+      additionalProperties: false
+      properties:
+        id:
+          type: string
+          format: uuid
+        eservice:
+          $ref: "#/components/schemas/DelegationEService"
+        delegate:
+          $ref: "#/components/schemas/DelegationTenant"
+        delegator:
+          $ref: "#/components/schemas/DelegationTenant"
+        contract:
+          $ref: "#/components/schemas/Document"
+        submittedAt:
+          type: string
+          format: date-time
+        rejectionReason:
+          type: string
+        state:
+          $ref: "#/components/schemas/DelegationState"
+      required:
+        - id
+        - eservice
+        - delegate
+        - delegator
+        - state
+    CompactDelegation:
+      type: object
+      additionalProperties: false
+      properties:
+        id:
+          type: string
+          format: uuid
+        eserviceName:
+          type: string
+        delegatedName:
+          type: string
+        delegatorName:
```

Comment:
Are we sure this is the only information the frontend needs?

Reply:
I think so, you can check the figma [here](https://www.figma.com/design/fqJMm6VdwMO41wbEj93wp0/Interop-%E2%80%94-v0.10-LLGG-2024?node-id=13263-29319&node-type=canvas&t=rhlVuk6CYk4esaMT-0) (do you have permissions to access it?)

Reply:
I don't have access but I requested it 👍